roles/digitalocean/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
Should we customize this file? I couldn't grasp it's idea. Is there a conflict between the licenses (Jarbas uses MIT)?